src/types.ts

```diff
@@ -17,30 +21,123 @@ export interface IReplaceableAccountId {
    */
   accountId: string | null;
 }
+
+/**
+ * See https://jmap.io/spec-core.html#get
+ */
 export interface IGetArguments<Properties extends IEntityProperties> extends IReplaceableAccountId {
   ids: string[] | null;
   properties?: (keyof Properties)[];
 }
 
+/**
+ * See https://jmap.io/spec-core.html#get
+ */
+export interface IGetResponse<Foo> {
+  accountId: string;
+  state: string;
+  list: Foo[];
+  notFound: string[];
+}
+
+/**
+ * See https://jmap.io/spec-core.html#changes
+ */
+export interface IChangesResponse {
+  accountId: string;
+  oldState: string;
+  newState: string;
+  hasMoreChanges: boolean;
+  created: string[];
+  updated: string[];
+  destroyed: string[];
+}
+
+/**
+ * See https://jmap.io/spec-core.html#set
+ */
 export interface ISetArguments<Foo> extends IReplaceableAccountId {
```

Comment:
You could use extends like for `IGetArguments<Properties extends IEntityProperties>`, to be more explicit about the type of Foo.